brave/brave-ios
# Fix #6683: remove wyre from on-ramp options

## Summary of Changes
Remove `Wyre` from the on-ramp options when user wants to purchase some assets.


This pull request fixes #6683

## Submitter Checklist:


## Test Plan:
1. Open Brave wallet
2. select different combination of tokens and network to see if `Wyre` is still an option for buying

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
